### DISCUSSION TITLE
Add HTML or Markdown or RichText to a slide?

### DISCUSSION THREAD

**Discussion by jsmarsch** [ShapeCrawler/ShapeCrawler#409] [Q&A]
I need to generate a slide that has some html that was input from another app.  It's really just formatting, and in fact it originates as markdown.  Would love an easy way of getting it into Powerpoint.  It doesn't have to remain as html, just need to preserve the formatting.

**Comment by AdamShakhabov** [comment]
If I correctly understood, you have a HTML page or a controller on the HTML-page and want to re-create this controller on PowerPoint slide, right?

Currently, I'm working on https://github.com/ShapeCrawler/ShapeCrawler/issues/53 issue where APIs like `IShapeCollection.AddAutoShape()`, `IShapeCollection.AddTable()` etc. are added to be able to create shape.
